projects/packages/search/src/customberg/components/sidebar/styles.scss

```
@@ -79,22 +81,25 @@
 
 	&.is-active:focus {
 		box-shadow: inset 0 0 0 var( --wp-admin-border-width-focus ) var( --wp-admin-theme-color ),
-			inset 0 0 -$border-width-tab 0 0 var( --wp-admin-theme-color );
+			inset 0 0 -$border-width-tab 0 var( --wp-admin-theme-color ); // stylelint-disable-line scss/operator-no-unspaced -- this is a sign, not an operation
```

Same as above. Also, you dropped a zero?
```suggestion
			inset 0 (-$border-width-tab) 0 0 var( --wp-admin-theme-color );
```

---

Ugh. Thanks.